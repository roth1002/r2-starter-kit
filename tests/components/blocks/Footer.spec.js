import React from 'react';
import { shallow } from 'enzyme';
import Component from 'components/blocks/Footer';

describe('<Footer />', () => {
  const initProps = {
  };

  let wrapper;
  beforeAll(() => {
    wrapper = shallow(<Component {...initProps} />);
  });

  it('should render initial component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
