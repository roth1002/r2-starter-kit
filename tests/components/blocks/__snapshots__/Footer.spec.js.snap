// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render initial component 1`] = `
<div
  className="default"
>
  <div>
    Icons from 
    <a
      href="https://www.flaticon.com/"
      title="Flaticon"
    >
      www.flaticon.com
    </a>
     is licensed by 
    <a
      href="http://creativecommons.org/licenses/by/3.0/"
      rel="noopener noreferrer"
      target="_blank"
      title="Creative Commons BY 3.0"
    >
      CC 3.0 BY
    </a>
  </div>
</div>
`;
